src/views/LoginView.vue

```diff
@@ -269,6 +269,10 @@ export default {
     scrollAndFocusLogin() {
       this.$refs.loginContainer.scrollIntoView();
       this.$refs.usernameInput.focus();
+    },
+    directToSignup() {
+      this.$router.push({ name: ROUTE_NAME_SIGNUP });
+      this.scrollAndFocusLogin();
```

Comment:
So I was unsure if I should do two different functions or if one would be better, 

```
    scrollAndFocusLogin(signup) {
      if(signup){
      this.$router.push({ name: ROUTE_NAME_SIGNUP });
      }
      this.$refs.loginContainer.scrollIntoView();
      this.$refs.usernameInput.focus();
    }, 
```